src/include/parser/pg_trigger.h

```diff
@@ -0,0 +1,81 @@
+#ifndef PG_TRIGGER_H
+#define PG_TRIGGER_H
+
+/* ----------------
+ *		compiler constants for pg_trigger
+ * ----------------
+ */
+#define Natts_pg_trigger 15
+#define Anum_pg_trigger_tgrelid 1
+#define Anum_pg_trigger_tgname 2
+#define Anum_pg_trigger_tgfoid 3
+#define Anum_pg_trigger_tgtype 4
+#define Anum_pg_trigger_tgenabled 5
+#define Anum_pg_trigger_tgisinternal 6
+#define Anum_pg_trigger_tgconstrrelid 7
+#define Anum_pg_trigger_tgconstrindid 8
+#define Anum_pg_trigger_tgconstraint 9
+#define Anum_pg_trigger_tgdeferrable 10
+#define Anum_pg_trigger_tginitdeferred 11
+#define Anum_pg_trigger_tgnargs 12
+#define Anum_pg_trigger_tgattr 13
+#define Anum_pg_trigger_tgargs 14
+#define Anum_pg_trigger_tgqual 15
```

Comment:
This file is copied from Postgres parser. We need to keep the macros the same to interpret the parsing tree generated by the Postgres parser. Actually, you don't have to review the newly imported head files under parser (like this one and `parsenodes.h`).